airflow/www/templates/airflow/dag.html

```diff
@@ -375,13 +375,13 @@ <h4 class="modal-title" id="dagModalLabel">
   {{ super() }}
   <script>
     function updateQueryStringParameter(uri, key, value) {
-      var re = new RegExp('([?&])' + key + '=.*?(&|$)', 'i');
+      var re = new RegExp(`([?&])${key}=.*?(&|$)`, 'i');
```

Comment:
What browsers support this? It's a "relatively" new syntax (and JS-in-html is not subject to any compile steps)

Reply:
Do you mean the template literals? Here is the information: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#browser_compatibility Only IE is marked "No support"

This should not be a concern to me though, because there is already usage of template literals in 2.0.0 ([example](https://github.com/apache/airflow/blob/2.0.0/airflow/www/templates/airflow/dags.html#L319) And I believe in some 1.10.* versions too) and so far there is not any complaint regarding this.

Let me know if you are referring to anything else?

Reply:
Ideally, we'd start migrating these scripts to external (compiled) files to make this concern moot. I'm not _super_ concerned about IE support, but I wish we had actual usage data to better inform us. Google Analytics appears to be installed on the documentation site—maybe its browser usage reports could serve as proxy (better than nothing)?

Reply:
I conferred w/ @ashb and we're both good on moving forward with this given it is already in practice. I'll add an issue for  migrating all of the inline scripting to external files.